<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="2" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-12-08T15:59:08Z" name="NopCommerce" finished-at="2021-12-08T15:59:37Z" duration-ms="29815">
    <groups>
    </groups>
    <test started-at="2021-12-08T15:59:08Z" name="Run on CHROME" finished-at="2021-12-08T15:59:33Z" duration-ms="25277">
      <class name="com.nopcommerce.user.Level_13_Register_Login_Log_Report">
        <test-method is-config="true" signature="beforeClass(java.lang.String, java.lang.String)[pri:0, instance:com.nopcommerce.user.Level_13_Register_Login_Log_Report@97e93f1]" started-at="2021-12-08T15:59:08Z" name="beforeClass" finished-at="2021-12-08T15:59:18Z" duration-ms="9986" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[CHROME]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[https://demo.nopcommerce.com/]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method signature="TC_01_Register()[pri:0, instance:com.nopcommerce.user.Level_13_Register_Login_Log_Report@97e93f1]" started-at="2021-12-08T15:59:18Z" name="TC_01_Register" finished-at="2021-12-08T15:59:28Z" duration-ms="10086" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertFalse(Assert.java:65)
at org.testng.Assert.assertFalse(Assert.java:75)
at commons.BaseTest.checkTrue(BaseTest.java:85)
at commons.BaseTest.verifyTrue(BaseTest.java:95)
at com.nopcommerce.user.Level_13_Register_Login_Log_Report.TC_01_Register(Level_13_Register_Login_Log_Report.java:83)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_01_Register -->
        <test-method signature="TC_02_Login()[pri:0, instance:com.nopcommerce.user.Level_13_Register_Login_Log_Report@97e93f1]" started-at="2021-12-08T15:59:28Z" name="TC_02_Login" finished-at="2021-12-08T15:59:32Z" duration-ms="4316" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_02_Login -->
        <test-method is-config="true" signature="afterclass()[pri:0, instance:com.nopcommerce.user.Level_13_Register_Login_Log_Report@97e93f1]" started-at="2021-12-08T15:59:32Z" name="afterclass" finished-at="2021-12-08T15:59:33Z" duration-ms="842" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterclass -->
      </class> <!-- com.nopcommerce.user.Level_13_Register_Login_Log_Report -->
    </test> <!-- Run on CHROME -->
    <test started-at="2021-12-08T15:59:08Z" name="Run on FIREFOX" finished-at="2021-12-08T15:59:37Z" duration-ms="29815">
      <class name="com.nopcommerce.user.Level_13_Register_Login_Log_Report">
        <test-method is-config="true" signature="beforeClass(java.lang.String, java.lang.String)[pri:0, instance:com.nopcommerce.user.Level_13_Register_Login_Log_Report@15a34df2]" started-at="2021-12-08T15:59:08Z" name="beforeClass" finished-at="2021-12-08T15:59:21Z" duration-ms="12918" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[FIREFOX]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[https://demo.nopcommerce.com/]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method signature="TC_01_Register()[pri:0, instance:com.nopcommerce.user.Level_13_Register_Login_Log_Report@15a34df2]" started-at="2021-12-08T15:59:21Z" name="TC_01_Register" finished-at="2021-12-08T15:59:32Z" duration-ms="11407" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertFalse(Assert.java:65)
at org.testng.Assert.assertFalse(Assert.java:75)
at commons.BaseTest.checkTrue(BaseTest.java:85)
at commons.BaseTest.verifyTrue(BaseTest.java:95)
at com.nopcommerce.user.Level_13_Register_Login_Log_Report.TC_01_Register(Level_13_Register_Login_Log_Report.java:83)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_01_Register -->
        <test-method signature="TC_02_Login()[pri:0, instance:com.nopcommerce.user.Level_13_Register_Login_Log_Report@15a34df2]" started-at="2021-12-08T15:59:32Z" name="TC_02_Login" finished-at="2021-12-08T15:59:36Z" duration-ms="3947" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_02_Login -->
        <test-method is-config="true" signature="afterclass()[pri:0, instance:com.nopcommerce.user.Level_13_Register_Login_Log_Report@15a34df2]" started-at="2021-12-08T15:59:36Z" name="afterclass" finished-at="2021-12-08T15:59:37Z" duration-ms="1518" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterclass -->
      </class> <!-- com.nopcommerce.user.Level_13_Register_Login_Log_Report -->
    </test> <!-- Run on FIREFOX -->
  </suite> <!-- NopCommerce -->
</testng-results>
